jest.setup.ts

```diff
@@ -2,12 +2,12 @@ import { config } from 'dotenv'
 import 'jest-styled-components' // include style rules in snapshots
 import fetchMock from 'jest-fetch-mock' // Mocks `fetch` calls in unittests
 
-// For simplicity, we will use CowSwap default .env for all projects and libs
-config({ path: __dirname + '/apps/cowswap-frontend/.env' })
-
 import { Readable } from 'stream'
 import { TextDecoder, TextEncoder } from 'util'
 
+// For simplicity, we will use CowSwap default .env for all projects and libs
+config({ path: __dirname + '/apps/cowswap-frontend/.env' })
```

Comment:
but are we sure about this? 

Feels wrong

Reply:
This is just a formatting change
![image](https://github.com/user-attachments/assets/32bb4d6b-b667-4dd7-9f27-17f5bc354cf9)

It was there already